purchase_request_to_rfq/models/stock.py

```diff
@@ -58,3 +57,9 @@ def do_transfer(self):
                     self._purchase_request_picking_confirm_message_content(
                         picking, request, requests_dict[request_id])
                 request.message_post(body=message, subtype='mail.mt_comment')
+
+    @api.multi
+    def do_transfer(self):
+        res = super(StockPicking, self).do_transfer()
+        self._purchase_request_reception_message()
```

Comment:
@zakiuu I would prefer a more meaningful name for the method. May be `_post_purchase_request_picking_confirm_message`

Reply:
@lmignon thanks for the review, done!